assets/src/data/services/apollo/queries/attendees/useAttendeesQuery.ts

```diff
@@ -0,0 +1,14 @@
+import { QueryHookOptions } from '@apollo/react-hooks';
+import { QueryResult } from '@apollo/react-common';
+
+import { useCacheQuery } from '@dataServices/apollo/queries';
+import { AttendeesList, AttendeesQueryArgs } from './types';
+import { EntityEdge } from '@dataServices/types';
+
+const useAttendeesQuery = <AttendeeEdge extends EntityEdge>(
+	queryOptions: QueryHookOptions<AttendeesList<AttendeeEdge>, AttendeesQueryArgs>
+): QueryResult<AttendeesList<AttendeeEdge>, AttendeesQueryArgs> => {
```

Comment:
a bit hard to read

Reply:
I agree, but it's the only way to lockdown types.